pywbem/tupleparse.py

```diff
@@ -1201,7 +1232,7 @@ def parse_property(tup_tree):
                 'EMBEDDEDOBJECT'],
                ['QUALIFIER', 'VALUE'])
 
-    quals = {}
+    quals = OrderedDict()
```

Comment:
Why OrderedDict rather than NocaseDict???

Reply:
The dict of the input arg is not used anyway; its items are put into a NocaseDict by the implementation of CIMProperty, so OrderedDict was the slightly cheaper alternative.

However, looking at the code again after your comment, it occurred to me that we now can pass in the list returned from list_matching() directly. Like we already do for CIMClass.

I'll change that.

Reply:
DONE